### PR TITLE
#2236 Manager's E2E cleanup script fails to do the cleanup

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #2236)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/scripts/handleE2eServices.js [hunk 1]
```diff
@@ -198,7 +198,9 @@ const deleteServices = async (services, apiRoot) => {
       const apiUrl = `${apiRoot}/${serviceType}`;
 
       await axios.put(`${apiUrl}/${serviceName}/stop`);
-      await utils.waitUntil({ condition: () => isServiceStopped(apiUrl) });
```
Comment (konekoya) [inline ohara-manager/scripts/handleE2eServices.js]: Wrong URL 😅

**Comment by konekoya** [comment]
retry manager-e2de

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
@wu87988622 hmm, still failing. But this time the broker was not able to be deleted. You can check the recent test report

**Comment by wu87988622** [comment]
retry manager-e2e

**Comment by wu87988622** [comment]
retry manager-e2e

**Comment by wu87988622** [comment]
retry manager-e2e

**Comment by wu87988622** [comment]
retry manager-e2e

**Comment by wu87988622** [comment]
retry manager-e2e

**Comment by wu87988622** [comment]
retry manager-e2e

**Comment by wu87988622** [comment]
retry manager-e2e

**Comment by wu87988622** [comment]
retry manager-e2e

**Comment by wu87988622** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

File: ohara-manager/client/cypress/e2e/homepage.test.js [hunk 2]
```diff
@@ -16,7 +16,7 @@
 
 import { PIPELINES, HOME } from '../../src/constants/urls';
 
-describe('HomePage', () => {
+describe.skip('HomePage', () => {
```
Comment (konekoya) [inline ohara-manager/client/cypress/e2e/homepage.test.js]: Remove the skip once the test is done.

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
It's now working again! Thanks for the help @wu87988622  :)

**Pull request merged**